test/lib/ufe/testDeleteCmd.py

```diff
@@ -138,3 +140,100 @@ def childrenNames(children):
 
         self.assertNotIn(sphereShapeName, sphereChildrenNames)
         self.assertNotIn(ball35Name, propsChildrenNames)
+
+        # undo to restore state to original.
+        cmds.undo()
+
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2011', 'testVisibility only available in UFE preview 2011 and greater')
```

Comment:
Ack!  Copy-paste damage.  'testVisibility' should read 'testDeleteArgs'.  Will fix along with any other code review comments.